svelte.config.js

```diff
@@ -3,6 +3,9 @@ import vercel from '@sveltejs/adapter-vercel';
 
 export default {
 	kit: {
-		adapter: process.env.VERCEL ? vercel() : netlify()
+		adapter: process.env.VERCEL ? vercel() : netlify(),
```

Comment:
not related to your change, but we could replace the two adapters with `adapter-auto`, which would do this automatically for us

Reply:
Makes sense; this line is just from the default `svelte.config.js`.